packages/ai/core/prompt/attachments-to-parts.ts

```diff
@@ -61,6 +73,18 @@ export function attachmentsToParts(attachments: Attachment[]): ContentPart[] {
               convertDataContentToUint8Array(base64Content),
             ),
           });
+        } else {
+          if (!attachment.contentType) {
+            throw new Error(
+              'If the attachment is not an image or text, it must specify a content type',
+            );
+          }
+
+          parts.push({
+            type: 'file',
+            data: convertDataContentToUint8Array(base64Content),
```

Comment:
this is unnecessary (also in the older code) and i saw hints that we use up cpu a lot. you can just pass in the base64